WalletWasabi/WebClients/Wasabi/WasabiClient.cs

```diff
@@ -256,7 +260,13 @@ public async Task<UpdateStatus> CheckUpdatesAsync(CancellationToken cancel)
 		{
 			var versions = await GetVersionsAsync(cancel);
 			var clientUpToDate = Constants.ClientVersion >= versions.ClientVersion; // If the client version locally is greater than or equal to the backend's reported client version, then good.
-			var backendCompatible = int.Parse(Constants.BackendMajorVersion) == versions.BackendMajorVersion; // If the backend major and the client major are equal, then our software is compatible.
+			var backendCompatible = int.Parse(Constants.ClientSupportBackendVersionMax) >= versions.BackendMajorVersion && versions.BackendMajorVersion >= int.Parse(Constants.ClientSupportBackendVersionMin); // If ClientSupportBackendVersionMin <= backend major <= ClientSupportBackendVersionMax, then our software is compatible.
+			var currentBackendMajorVersion = versions.BackendMajorVersion.ToString();
+			if (CurrentBackendMajorVersion != currentBackendMajorVersion)
```

Comment:
```suggestion
			if (backendCompatible && CurrentBackendMajorVersion != currentBackendMajorVersion)
```

Reply:
I need to detect the change here. Even if it is not compatible I want to see the current version on the About tab.

Reply:
You shouldn't use the current version if it's not compatible with the current version. Don't bother with the about tab IMO.

Reply:
It should be handled by `WasabiClient` with an `isBackendCompatible` property, not by blocking the update of the `CurrentBackendMajorVersion` property.

Reply:
`using var response = await TorClient.SendAsync(HttpMethod.Get, $"/api/v{WasabiClient.CurrentBackendMajorVersion}/btc/chaumiancoinjoin/states/").ConfigureAwait(false);` will send out old request to new backend.

Reply:
I don't get how your fix related to that. Let's wait for Lucas's addition to this PR, that will handle the compatibility IMO. CurrentBackendMajorVersion should always reflect the current version even if it is not compatible.

Reply:
This is a huge bug. @lontivero can you help me out here? I must be explaining it wrong.

Reply:
![image](https://user-images.githubusercontent.com/9156103/83112155-053bdf00-a0c6-11ea-8bdc-eea94c8d8c51.png)

Reply:
We are going to release a new client version with this code that has the `CurrentBackendMajorVersion = "3"` initially so, it doesn't matter what version the backend is running the client will make a request to the v3 API. This while happen until the client gets the API version running by the coordinator, after that it will use that version. 
This means that if we release this code it will work perfectly well. However, once we update the API version to v4 the client will perform at least one request to v3 API during initialization.

Reply:
@nopara73 I addressed your concerns. It is fixed.